tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -170,6 +170,8 @@ internal static partial class ConfigurationKeys
         /// </summary>
         public const string GlobalSamplingRate = "DD_TRACE_SAMPLE_RATE";
 
+        public const string RareSamplerEnabled = "DD_APM_ENABLE_RARE_SAMPLER";
```

Comment:
Why not `DD_TRACE_ENABLE_RARE_SAMPLER`? Are we trying to match an existing setting in the trace agent? Are all tracer libraries using this name?

Reply:
That's the name as defined in the RFC, so I suppose it will be the same for other tracers 🤔 I do agree that DD_TRACE_ENABLE_RARE_SAMPLER would be more consistent.

Reply:
Ok, if it's in the RFC 🤷🏽‍♂️. Thanks.

Reply:
Did you mean [this RFC](https://docs.google.com/document/d/1wdl6yBXtvR1jwldzrRcmeuhATWfbvRzaCkVf1LKOiLE/edit?usp=sharing)? It seems to cover only the trace agent. I don't see anything specific about tracers except:
> We might have a mechanism for rare traces in the .NET tracer. To be clarified

Is there a separate RFC that covers moving these features (status, samplers, etc) from agent to tracer? Wouldn't `DD_APM_ENABLE_RARE_SAMPLER=true` on a host potentially enable this feature on both the agent and the tracer?

Reply:
`DD_APM_ENABLE_RARE_SAMPLER` but shouldn't we use the same parameter? I mean if we're enabling stats computation, the agent shouldn't be impacted here. If we don't, we wouldn't be impacted?

Reply:
I find it confusing because all trace agent setting being with `DD_APM_*`. Most tracer settings are `DD_TRACE_*` and _none_ are `DD_APM_*`.